### PR TITLE
Fix strikethrough in docs

### PR DESCRIPTION
**Pull request by imurchie**
## Proposed changes

Fix the strikethrough in the xcuitest migration docs.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by dpgraham — APPROVED** [review]
Does this fix the strikethrough problems in MkDocs?

File: docs/en/advanced-concepts/migrating-to-xcuitest.md [hunk 1]
```diff
@@ -98,12 +98,12 @@ In the previous UIAutomation-based driver, if you tried to click on an element t
 
 Finally, a list of known issues with the initial 1.6 release (we'll strike through issues which have been resolved):
```
Comment (SrinivasanTarget) [inline docs/en/advanced-concepts/migrating-to-xcuitest.md]: Can we update as `1.7`?

Comment (imurchie) [reply docs/en/advanced-concepts/migrating-to-xcuitest.md]: We can. Not in this PR, though.

**Comment by imurchie** [comment]
It does. `~~` needs a markdown extension for MKDocs. This is easier, and works in Github and MKDocs.

File: docs/en/advanced-concepts/migrating-to-xcuitest.md (hunk 1 above)
Comment (SrinivasanTarget) [reply docs/en/advanced-concepts/migrating-to-xcuitest.md]: Ok

**Review by SrinivasanTarget — APPROVED** [review]

**Pull request merged**